tools/start_all_tmux.sh

```diff
@@ -50,6 +50,7 @@ declare -a imageclassification=("IMC" "$(pwd)/../lucida/djinntonic/imc/")
 declare -a digitrecognition=("DIG" "$(pwd)/../lucida/djinntonic/dig/")
 declare -a facerecognition=("FACE" "$(pwd)/../lucida/djinntonic/face")
 declare -a weather=("WE" "$(pwd)/../lucida/weather")
+declare -a musicservice=("MS" "$(pwd)/../lucida/musicservice")
 
 declare -a services=(
     commandcenter
```

Comment:
@zhexuanc, need to add music service here as well to able to create the window for the music service.